### ISSUE TITLE
Contribution: checkpoints AVAILABLE!

### ISSUE THREAD

**Issue by freds0** [mindslab-ai/nuwave#18]
Hi guys, 
First I would like to thank @junjun3518  for the excellent work of developing and sharing the code. I trained the model following the paper settings for two weeks on a V100 GPU using ratio=2 and 3. I would like to contribute to the project by sharing the checkpoints. Below are the download links.

**nuwave x2**:
https://drive.google.com/file/d/1pegayKs-i78yWlPuLIp-BCU8KxxCpBzd/view?usp=sharing

**nuwave x3**:
https://drive.google.com/file/d/12RUMjEALAs0EoEw6Fqf9ZkpTm3COX6sf/view?usp=sharing

The following are images of the training logs. 

**nuwave x2**:

_epoch_:
![epoch](https://user-images.githubusercontent.com/6883434/162975411-473ad06a-d3dd-4d09-8119-67d0750c71a1.svg)

_loss_:
![loss](https://user-images.githubusercontent.com/6883434/162975425-9d5f024f-c5c5-438b-98c9-72058fe80fd9.svg)

_val loss_:
![val_loss](https://user-images.githubusercontent.com/6883434/162975434-fbe707dd-59c1-4de5-8d81-3efd1cb8b72b.svg)

**nuwave x3**:

_epoch_:
![epoch](https://user-images.githubusercontent.com/6883434/162975564-fdc7cfd2-0f38-494a-993a-35203ec4a431.svg)

_loss_:
![loss](https://user-images.githubusercontent.com/6883434/162975489-a828d243-6871-40fc-ad3f-7801137a1e39.svg)

_val loss_:
![val_loss](https://user-images.githubusercontent.com/6883434/162975495-ad6226ea-9504-4eda-b59a-c737e4a0a321.svg)

b-a01c-8df0b99c9e0e.svg)

**Comment by freds0** [comment]
I also ran the test scripts:

**nuwave_x2:**

![Captura de tela_2022-04-12_10-49-08](https://user-images.githubusercontent.com/6883434/162977286-2c9558b1-db5f-4f4f-8108-a34fa34a77a0.png)

**nuwave_x3:**

![Captura de tela_2022-04-12_10-48-49](https://user-images.githubusercontent.com/6883434/162977324-19952108-2952-4438-b7b9-d6e157619917.png)

**Comment by junjun3518** [comment]
Thank you for your great contribution! I add link of this issue on [README.md](https://github.com/mindslab-ai/nuwave/blob/master/README.md)!

**Comment by nateraw** [comment]
Hey there @junjun3518 and and @freds0. Can I (or @junjun3518, if they'd like) share this model on the [Hugging Face Hub](https://hf.co/models)?

**Comment by psp0001060** [comment]
Thank you for your great contribution! I want to use checkpoint to upsample music. What performance devices do I need?  Can a personal laptop run this model?

**Comment by junjun3518** [comment]
Yes, it is also runnable with CPU (but very slow).
For the music case, I don't recommend you to use this project since it is only trained with clean speech without music

**Comment by psp0001060** [comment]
> Yes, it is also runnable with CPU (but very slow). For the music case, I don't recommend you to use this project since it is only trained with clean speech without music

Thanks for getting back to me so quickly. If I use music data to train the model, and then use the model to upsample, is that feasible?

**Comment by junjun3518** [comment]
I think that it is different from the instrumental of the source.
For example, it is hard to apply to electronic music, since high-frequency sounds of electronic do not correlate with low-frequency sounds.
For classic instrumental or piano, it is applicable.

**Comment by saurabh-kataria** [comment]
Would you say 700 epochs are enough for training?
